randovania/games/am2r/json_data/Hydro Station.txt

```diff
@@ -38,8 +38,14 @@ Extra - map_name: rm_a2h02
   * Open Passage to Hydro Station Entrance/Dock to Hydro Station Exterior
   > Dock to Broken Save Station
       Trivial
+  > Dock to Arachnus Save Station
+      Any of the following:
+          Space Jump Wall
+          Speed Booster and Shinesparking Tricks (Beginner)
   > Pickup (Missile Tank)
-      Can Use Spider Ball
+      Any of the following:
+          Can Use Spider Ball
+          Speed Booster and Shinesparking Tricks (Beginner)
```

Comment:
this is an unnecessary connection. arachnus save station already has the same requirement for shinespark, and the dock to entrance has the requirement via spider ball